extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/tuple/ArrayOfDoublesSketchOperations.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.datasketches.tuple;
+
+import com.google.common.base.Charsets;
+import com.yahoo.memory.Memory;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.yahoo.sketches.tuple.ArrayOfDoublesAnotB;
+import com.yahoo.sketches.tuple.ArrayOfDoublesCombiner;
+import com.yahoo.sketches.tuple.ArrayOfDoublesIntersection;
+import com.yahoo.sketches.tuple.ArrayOfDoublesSketch;
+import com.yahoo.sketches.tuple.ArrayOfDoublesSketches;
+import com.yahoo.sketches.tuple.ArrayOfDoublesUnion;
+
+import io.druid.java.util.common.IAE;
+import io.druid.java.util.common.ISE;
+
+import com.yahoo.sketches.tuple.ArrayOfDoublesSetOperationBuilder;
+
+public class ArrayOfDoublesSketchOperations
+{
+
+  public enum Func
```

Comment:
For consistency with the enclosing class, suggested to call this enum "Operation".

Reply:
ok